backend/spec/fixtures/snapshots/api/v1/user.json

```diff
@@ -1,26 +1,26 @@
 {
   "data": {
-    "id": "d9542584-bc72-4edf-9095-9d44d190b85e",
+    "id": "6f7aa668-8574-4e63-b9d9-416b028f7f9f",
     "type": "user",
     "attributes": {
       "first_name": "Dawna",
       "last_name": "Block",
       "email": "user@example.com",
       "role": "light",
+      "created_at": "2022-10-13T12:52:31.345Z",
+      "ui_language": "en",
+      "account_language": null,
+      "otp_required_for_login": false,
       "confirmed": true,
       "approved": null,
       "invitation": null,
-      "created_at": "2022-06-24T09:06:19.915Z",
       "owner": false,
       "avatar": {
+        "analyzed": false,
         "small": null,
         "medium": null,
-        "original": null,
-        "analyzed": false
-      },
-      "ui_language": "en",
-      "account_language": null,
-      "otp_required_for_login": false
+        "original": null
```

Comment:
I don't think it was necessary to change all these json. Could we throw away changes which are not needed? Or keep it now, but try to prevent it future? :)

Reply:
OK, I'll try to use the CONSERVATIVE snaphost update, in general specs were giving me a lot of troubles (and taking forever to run) and I guess that's why I gave up, replacing snaphots spared me at least some troubles :)

Reply:
Yeah, it is slow :), which I think is quite common when using factory bot. We can either use `before_all` tweak which Tomasz uses sometimes, but it has its own cons --> when used, you need to put all variables into it which can get quite messy for files with a lot of tests. When it comes to speed, I personally prefer fixtures which are so much faster (and minitest).

Reply:
OK, I managed to reduce the number of changes to snapshots 😅